common-tomcat-maven-plugin/src/test/java/org/apache/tomcat/maven/common/TomcatManagerTest.java

```diff
@@ -74,7 +74,7 @@ protected void setUp()
         Context context = tomcat.addContext( "", System.getProperty( "java.io.tmpdir" ) );
         uploadServlet = new UploadServlet();
         tomcat.addServlet( context, "foo", uploadServlet );
-        context.addServletMapping( "/*", "foo" );
+        context.addServletMappingDecoded( "/*", "foo" );
```

Comment:
The javadocs don't give me much of a reason. The missing parameter is `jspWildcard` – true if name identifies the JspServlet and pattern contains a wildcard; false otherwise and it's set to `false` in this 2 parameter variant of the method.